bots/kook/message.py

```
@@ -243,7 +243,7 @@ async def fetch_target_list(target_list: list) -> List[Bot.FetchedSession]:
         return lst
 
     @staticmethod
-    async def post_message(module_name, message, user_list: List[Bot.FetchedSession] = None, i18n=False, **kwargs):
+    async def post_message(module_name, message, user_list=[], i18n=False, **kwargs):
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)